ppr-ui/src/components/tables/common/TableRow.vue

```diff
@@ -560,17 +561,23 @@ export default defineComponent({
           window.document.body.appendChild(a)
           a.setAttribute('style', 'display: none')
           a.href = url
-          // Format: [Date (in YYYY-MM-DD format)] BCPPR [Two Letter Registration Type Code
+          // Format (PPR): [Date (in YYYY-MM-DD format)] BCPPR [Two Letter Registration Type Code
           // (only for Standard Registrations)] [Verification Statement Type] - [Registration Number]
           // Example: 2022-01-03 BCPPR SA Registration Verification - 100559B
+
+          // Format (MHR): [Date (in YYYY-MM-DD format)] BCMHR [Registration || Transfer] - [Registration Number]
+          // Example: 2023-03-13 BCMHR Registration - 150378
+
           const today = new Date()
+          let regTypeString = props.isPpr ? '_BCPPR_' : '_BCMHR_Registration_'
+          regTypeString = isMhrTransfer(item) ? '_BCMHR_Transfer_' : regTypeString
```

Comment:
You could* simplify this but i'm not gonna be too picky.
Maybe take a quick look at this and consider how it might be simplified, up to you ;)

Reply:
Simplified :)